docker-compose.yml

```diff
@@ -1,9 +1,9 @@
 version: '3'
 services:
-  nginx_web:
-    image: httpd:latest
-    container_name: httpd_web
+  nginx_proxy:
+    image: nginx:latest
+    container_name: nginx_proxy
     volumes:
-    - ./public/:/usr/local/apache2/htdocs/
+    - ./config/nginx/proxy.conf:/etc/nginx/conf.d/default.conf
     ports:
     - 80:80
```

Comment:
This should be the only container with `80:80` (port 80 exposed externally). All others should just be `80` (internal only).